src/Microsoft.Health.Dicom.Core.UnitTests/Extensions/DicomTagExtensionsTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Dicom;
+using Microsoft.Health.Dicom.Core.Extensions;
+using Xunit;
+
+namespace Microsoft.Health.Dicom.Core.UnitTests.Extensions
+{
+    public class DicomTagExtensionsTests
+    {
+        [Fact]
+        public void GivenValidDicomTag_WhenGetPath_ThenShouldReturnCorrectValue()
+        {
+            Assert.Equal("0014408B", DicomTag.UserSelectedGainY.GetPath());
+        }
+
+        [MemberData(nameof(MemberDataForTestingGetDefaultVR))]
+        [Theory]
```

Comment:
>[Theory] [](start = 8, length = 8)

nit: I have typically seen the `[Theory]` and `[Fact]` attributes on top, but this may be the style of the repo

Reply:
it's perfect if there is some tools for this. :)
Updating...